jdaviz/configs/default/plugins/export/export.py

```diff
@@ -341,11 +331,32 @@ def export(self, filename=None, show_dialog=None, overwrite=False,
                     raise FileExistsError(f"{filename} exists but overwrite=False")
                 return
 
+            # temporarily "clean" incompatible marks of unicode characters, etc
+            restores = []
+            for mark in viewer.figure.marks:
+                restore = {}
+                if len(getattr(mark, 'text', [])):
+                    if not isinstance(mark, ShadowMixin):
+                        # if it is shadowing another mark, that will automatically get updated
+                        # when the other mark is restored, but we'll still ensure that the mark
+                        # is clean of unicode before exporting.
+                        restore['text'] = [t for t in mark.text]
+                    mark.text = [t.strip() for t in mark.text]
+                if len(getattr(mark, 'labels', [])):
+                    restore['labels'] = mark.labels[:]
+                    mark.labels = [lbl.strip() for lbl in mark.labels]
+                restores.append(restore)

```

Comment:
I'm not sure I follow - do you mean store a copy of the original mark?  I'm trying to store/restore as little as possible to minimize memory impacts.  I tried to avoid the append, but there can be some copying-referencing issues if we're not careful then.

Reply:
```suggestion
                if restore != {}:
                    restores.append(restore)
                    restore_marks.append(mark)
```

Would `restore_mark` have a copy of the object or a reference to the original in this case? I can never keep track of when python does which, but this would avoid looping over things that don't need to be restored. If memory is an issues it can stay as-is, I honestly don't think this is going to be impactful either way here.

Reply:
I suspect this would be more expensive than looping over the empty dictionary, but I'm not sure.  We could also track indices, but I'm not sure its worth it either 🤷‍♂️

Reply:
As I said, I don't feel strongly about it, I'll just approve.